apps/jobs/models.py

```diff
@@ -102,6 +102,11 @@ class Submission(TimeStampedModel):
         null=True,
         blank=True,
     )
+    code_upload_env_output_file = models.FileField(
+        upload_to=RandomFileName("submission_files/submission_{id}"),
+        null=True,
+        blank=True,
+    )
```

Comment:
Can you rename this to `code_upload_environment_log_file` and make the required changes to all relevant places